proto/settlus/oracle/v1alpha1/oracle.proto

```diff
@@ -67,7 +66,7 @@ message RoundInfo {
   uint64 id = 1;
   int64 prevote_end = 2;
   int64 vote_end = 3;
-  repeated OracleData oracle_data = 4;
+  repeated string ownerships = 4;
```

Comment:
Please leave `repeated OracleData oracle_data`. This breaks state compatibility and we might should rollback it again in the future.